.github/workflows/chromatic.yml

```diff
@@ -62,7 +62,8 @@ jobs:
         with:
           fetch-depth: 0 # 👈 Required to retrieve git history
 
-      - name: Check for changes
+      - name: Check for changes (push)
+        if: ${{ github.event_name == 'push' }}
         uses: dorny/paths-filter@v2
         id: filter
         with:
```

Comment:
wait for the filters do we need package json? asking cuz of the failure [here](https://github.com/isomerpages/isomercms-frontend/actions/runs/6247004407/job/16958757263#step:7:23), but not 100% sure if needed

Reply:
Yup I think that's a good idea, added in 48e10d7!